pkg/build/node/cni.go

```diff
@@ -37,11 +37,6 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: kindnet
-  annotations:
-    seccomp.security.alpha.kubernetes.io/allowedProfileNames: docker/default
-    seccomp.security.alpha.kubernetes.io/defaultProfileName: docker/default
-    apparmor.security.beta.kubernetes.io/allowedProfileNames: runtime/default
-    apparmor.security.beta.kubernetes.io/defaultProfileName: runtime/default
```

Comment:
we seem to have another PSP here for kindnet.
with the addition of the permissive global PSP i don't think this will be needed.

Reply:
if users change the global PSP maybe this will break, I will keep it, only removing those entries because of the apparmor problems mentioned in other comment

Reply:
interestingly calico for example does not seem to ship with a PSP

Reply:
I've copied pasted from flannel IIRC